### ISSUE TITLE
A calendar widget - Disabled dates and Use of color

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/wcag#2265]
Question for the WCAG experts.

If a calendar widget has a disabled date range, dates are not available for selection, and the disabled dates are a grey background, is this a failure for 1.4.1: Use of Color?

The date cell includes:
- The date numbers meet contrast ratio requirements
- ARIA aria-disabled true is set on the dates

**Comment by JAWS-test** [comment]
The disabled and non-disabled dates should be visually distinguishable. If only their background color differs, then the contrast ratio between the background colors should be 3:1 or more, so as not to violate 1.4.1.
The text contrasts of the disabled dates do not have to be sufficient, i.e. can be less than 4.5:1.

`disabled` is better than `aria-disabled`, because otherwise the disabled status may no longer be visible when using the high contrast mode in Windows.

**Comment by LaurenceRLewis** [comment]
So the sufficient contrast between disabled and active dates background causes it not to be considered a 1.4.1 Use of colour fail. Have I understood that correctly @JAWS-test?

Thanks for the disabled attribute tip.

**Comment by JAWS-test** [comment]
In 1.4.1 there is the exception that color coding is ok if the contrast ratio between the colors is 3:1 and the concrete colors have no meaning in themselves (as in green = good, red = bad)

**Comment by JAWS-test** [comment]
Of course, it is always useful to use another distinguishing feature besides colors (e.g. font style), but it is not required
